spec/lib/api/v3/work_packages/work_package_representer_spec.rb

```diff
@@ -406,17 +404,11 @@
 
       describe 'priority' do
         let(:priority) { work_package.priority }
-        let(:link) { "/api/v3/priorities/#{priority.id}".to_json }
-        let(:title) { "#{priority.name}".to_json }
-        let(:href_path) { '_links/priority/href' }
-        let(:title_path) { '_links/priority/title' }
-
-        it 'has a link' do
-          is_expected.to be_json_eql(link).at_path(href_path)
-        end
 
-        it 'has a title' do
-          is_expected.to be_json_eql(title).at_path(title_path)
+        it_behaves_like 'has a titled link' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.